docs/sdk-and-tools/indices/transactions.md

```diff
@@ -0,0 +1,81 @@
+---
+id: transactions
+title: transactions
+---
+
+The `_id` field for this index is composed of transaction hash hex encoded.
+(example: `cad4692a092226d68fde24840586bdf36b30e02dc4bf2a73516730867545d53c`)
+
+| Field             | Description                                                                                                                            |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| miniBlockHash     | The miniBlockHash represents the hash of the miniblocks in which the transaction is included.                                          |
+| nonce             | The nonce field represents sequence number of transactions sent from a given address.                                                  |
+| round             | The round field represents the round of block when transaction is executed.                                                            |
+| value             | The value field represents the amount of EGLD from the sender to the receiver.                                                         |
+| receiver          | The receiver field represents the destination address of the transaction.                                                              |
+| sender            | The sender field field represents the address of the transaction sender.                                                               |
+| receiverShard     | The receiverShard field represents the shard ID of the receiver address.                                                               |
+| senderShard       | The senderShard field represents the shard ID of the sender address.                                                                   |
+| gasPrice          | The gasPrice field represents the price of the gas.                                                                                    |
+| gasLimit          | The gasLimit field represents the limit of the amount of EGLD the sender is willing to pay for the transaction.                        |                                                                  |
+| gasUsed           | The gasUsed field represents the amount of gas used by the transaction.                                                                |
+| fee               | The fee field represents the amount of EGLD the sender paid for the transaction.                                                       |
+| initialPaidFee    | The initialPaidFee field represents the initial amount of EGLD the sender paid for the transaction, before the refund.                 |
+| data              | The data field holds additional information for a transaction. It can either contain a simple message/ESDT related information/etc.    |
```

Comment:
It can contain a simple message, a function call, an ESDT transfer payload, or so on